react-graphql-apollo-tutorial.md

```diff
@@ -1145,26 +1147,22 @@ const RepositoryItem = ({ ... }) => (
 );
 {{< /highlight >}}
 
-When you star a repository with the previous implementation, the "Star" button disappears. That's great, because it means that the `viewerHasStarred` boolean has been updated in Apollo Client's cache for the identified repository. There was no extra implementation needed from your side. Apollo Client was able to match the mutation result with the repository identifier to the repository entity in Apollo Client's cache. The props were updated and the UI re-rendered.
-
-Yet on the other side the count of stargazers who have starred the repository isn't updated. It's because it cannot be retrieved from GitHub's API. That's why it would be up to you to update the count yourself in Apollo Client's cache. No worries for now, you will find out more about this topic in one of the following sections.
+When you star a repository as above, the "Star" button disappears. This is what we want, because it means the `viewerHasStarred` boolean has been updated in Apollo Client's cache for the identified repository.  Apollo Client was able to match the mutation result with the repository identifier to the repository entity in Apollo Client's cache, the props were updated, and the UI re-rendered. However, the count on the clients side hasn't updated, because it can't be retrieved from GitHub's API. It needs update the count  in Apollo Client's cache, which we will cover in the next sections.
```

Comment:
I find the previous explanation with the two paragraphs clearer than the new version. 

"However, the count on the clients side hasn't updated, because it can't be retrieved from GitHub's API. It needs update the count  in Apollo Client's cache, which we will cover in the next sections." 

to 

Yet on the other side the count of stargazers who have starred the repository isn't updated. It's because it cannot be retrieved from GitHub's API. That's why it would be up to you to update the count yourself in Apollo Client's cache. No worries for now, you will find out more about this topic in one of the following sections.

Reply:
It's not.  I am a native English speaker, and I had to read the original paragraph three times to understand what you meant.  But I will change it back.

Reply:
I changed it to this:
"Yet, on the other side, the count of stargazers who have starred the repository isn't updated because it cannot be retrieved from GitHub's API. The count must be updated the count in Apollo Client's cache. You will find out more about this topic in one of the following sections."

Again, we must remove irrelevant phrases.  Telling the reader it's up to them to do something themselves isn't helpful.  They just need to know it has to be done.  The phrase "no worries for now" is too informal for its technical surroundings.

Reply:
I like the new version of it! Thanks for improving it one more time 👍 

Good tip about the informal wording 👍